src/Microsoft.VisualStudio.Editors/ResourceEditor/ResourceListView.vb

```diff
@@ -1132,7 +1132,7 @@ Namespace Microsoft.VisualStudio.Editors.ResourceEditor
             Else
                 Try
                     ThumbnailSourceImage = Resource.ResourceTypeEditor.GetImageForThumbnail(Resource, Me.BackColor)
-                Catch ex As Exception When Common.Utils.ReportWithoutCrash(ex, NameOf(GetThumbnailIndex), NameOf(ResourceListView))
```

Comment:
The removed ReportWithoutCrash callsites in this file account for 300,000+ fault events :)